IMSCovidTracker/Views/HomePage.xaml.cs

```diff
@@ -24,57 +18,17 @@ public HomePage()
 
             // where to display tutorial
             navbarComponent.AbsoluteLayoutElement = homeLayout;
-
-            _viewModel.WidgetCollectionChanged += _viewModel_WidgetCollectionChanged;
-
-        }
-
-        private void _viewModel_WidgetCollectionChanged(object sender, int delay)
-        {
-            _ = ForceCollectionLayout(delay);
         }
 
         protected override void OnAppearing()
         {
             base.OnAppearing();
-            
-            // CollectionView in iOS is a bit buggy but forcing layout after rendering seems to solve some issues
-            //if (Device.RuntimePlatform == Device.iOS)
-            ForceLayout();
 
             // show only in first launch
             if (VersionTracking.IsFirstLaunchEver)
             {
                 _ = navbarComponent.ShowTutorial();
             }
         }
-
-        /// <summary>
-        /// <i>CollectionLayout is a bit buggy when adding/removing items so we must force some kind of re-layout</i> <br/>
-        /// Force layout for collectionview
-        /// </summary>
-        /// <returns></returns>
-        private async Task ForceCollectionLayout(int delay = 400)
```

Comment:
Didn't you get display bugs while Adding/Removing widgets here ? I mean, the reason i'm doing this is related to the Issue you opened with Xamarin.Forms where the Spacing between the items breaks when you add/remove items

Reply:
I changes it to not use spacing and instead sets small margin on each item for spacing. It's not ideal but it looks almost the same and you don't need to call layout and calculate height